hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/Server.java

```diff
@@ -3158,6 +3161,9 @@ private void requeueCall(Call call)
         throws IOException, InterruptedException {
       try {
         internalQueueCall(call, false);
+        long deltaNanos = Time.monotonicNowNanos() - call.timestampNanos;
```

Comment:
Isn't it the same time as ENQUEUE calculated in `internalQueueCall(call, false)` of the before line?

Reply:
Hi, @tasanuma , thanks a lot for reviewing.  yes, it is the same time when calculated in internalQueueCall(call, false).
But in fact， this time is REQUEUE time , original implementation does not record this time.